### DISCUSSION TITLE
Allowing free text on a multiple choice question

### DISCUSSION THREAD

**Discussion by Roialfassi** [heyform/heyform#27] [Ideas]
Is there a feature like the other in google forms

**Comment by dearroy** [comment]
@Roialfassi Could you please include a screenshot?

**Comment by Roialfassi** [comment]
These options are available on both google forms and Qualtrics for describing an option that is not on these multiple options but still to know what it is.

![Option1](https://github.com/heyform/heyform/assets/26362878/953a6833-353a-413a-9a94-c82452283921)
![Option2](https://github.com/heyform/heyform/assets/26362878/01e8e1b2-ab18-4305-acd9-f2bd30ad752a)

**Reply by dearroy** [reply]
OK, I got you! I will put this on the planning list.
